libraries/standard/mqtt/src/mqtt.c

```diff
@@ -313,7 +340,7 @@ static int32_t recvExact( const MQTTContext_t * pContext,
     uint8_t * pIndex = NULL;
     size_t bytesRemaining = bytesToRecv;
     int32_t totalBytesRecvd = 0, bytesRecvd;
-    uint32_t entryTimeMs = 0U;
+    uint32_t entryTimeMs = 0U, elapsedTime = 0U;
```

Comment:
What units is `elapsedTime` in?  From line 383, it looks like milliseconds so let's name it with an `Ms` suffix so that it's clear to the reader.

Reply:
Added